Examples/SubscriberExample/Sources/Screens/Map/MapViewController.swift

```diff
@@ -311,6 +311,6 @@ extension MapViewController: SubscriberDelegate {
     }
     
     func subscriber(sender: Subscriber, didUpdateDesiredInterval interval: Double) {
-        locationUpdateInterval = interval
+        locationUpdateInterval = interval / 1000
```

Comment:
Why do we have to do this conversion here? The `interval` is already in milliseconds and we use the `locationUpdateInterval` only in the location animator which expects to get milliseconds. 

What's more, [the initial value](https://github.com/ably/ably-asset-tracking-swift/blob/b3e4817edf63a893ae457757637a14f8560576ed/Examples/SubscriberExample/Sources/Screens/Map/MapViewController.swift#L53) of the `locationUpdateInterval` is set to `resolution.desiredInterval` which is in milliiseconds, but in this line we're saving it in seconds 😅 

I think that we should simply store the `interval` in milliseconds and get rid of the conversion, WDYT? 😉

Reply:
I think the conversion was probably done so that we can store the interval as a `TimeInterval`, which is a built-in type for representing durations. But I think you're right, might be neater without the conversion, will change it.

But in #378 I've said that I think we should switch to using `TimeInterval` for durations throughout the SDK.

Reply:
If the `TimeInterval` is a standard in iOS I'd say go for it 😉 But on Android, I'm not aware of such a thing, so we would have a slight difference between platforms, but I guess it would be okay, right @QuintinWillison ? 🤔

Reply:
Resolving the conversation as it was fixed in https://github.com/ably/ably-asset-tracking-swift/pull/379/commits/5a4db283d68f211dded51bc0e4c531763e02e9ae